libs/libskynet/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+	"printWidth": 120,
+	"useTabs": true,
```

Comment:
It sounds like it's a personal preference but tabs in code produce indentation of 8 chars in github instead of 2 when they are regular spaces. It's also not consistent with any other repository that we have.

Reply:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference